src/ducks/settings/TagsList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import Fuse from 'fuse.js'
+import orderBy from 'lodash/orderBy'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import TagListItem from 'ducks/settings/TagListItem'
+
+const fuse = new Fuse([], {
+  findAllMatches: true,
+  threshold: 0.3,
+  keys: ['label']
+})
+
+const listStyle = { padding: '.5rem 0' }
+
+const TagsList = ({ tags, filter, sort }) => {
+  const { isMobile } = useBreakpoints()
+  const [sortKey, sortOrder] = sort.split('-')
+  let filteredTags = tags
+  if (filter.length !== 0) {
+    fuse.setCollection(filteredTags)
+    filteredTags = fuse.search(filter).map(result => result.item)
```

Comment:
Be careful, with that you're looking for exact string. So if you create a Tag "Été 2022" you will never find it by writing "ete 2022". I think we should handle that case since on mobile it can be difficult. 

There is a lot of SO answers about how to do that ;)